charts/aac-manage-case-assignment/values.preview.template.yaml

```diff
@@ -5,13 +5,22 @@ java:
   keyVaults:
     aac:
       secrets:
-        - aac-manage-case-s2s-secret
-        - idam-mca-client-secret
-        - idam-mca-system-user-username
-        - idam-mca-system-user-password
-        - idam-noc-approver-username
-        - idam-noc-approver-password
-        - notify-mca-api-key-secret
+        - name: AppInsightsInstrumentationKey
+          alias: azure.application-insights.instrumentation-key
```

Comment:
```suggestion
```
Similar to the comment on the Def Store PR, we only need the "essentials" for preview.